### DISCUSSION TITLE
Add InfoWindowClicked to Mvvm implementation

### DISCUSSION THREAD

**Discussion by davefxy** [VladislavAntonyuk/MauiSamples#186] [Q&A] [answered]
To provide the InfoWindow click event to the mvvm version of map
Make the following change to MainPage.xaml 👍 
`					<mauiMaps:CustomPin
                        InfoWindowClicked="Pin_InfoWindowClicked"
						Location="{Binding Location}"
						Address="{Binding Address}"
						Label="{Binding Description}"
						ImageSource="{Binding ImageSource}"
						Type="Place"/>
`
change to MainPage.xaml.cs:
`    private async void Pin_InfoWindowClicked(object sender, PinClickedEventArgs e)
    {
        await Toast.Make("Info Window is clicked").Show();
    }
`

**Comment by VladislavAntonyuk** [comment, marked as answer]
MVVM uses commands instead of events.
I suggest you use EventToCommand from ComminutyToolkit.Maui
